tests/cypress/e2e/accessibility/profile.cy.js

```diff
@@ -12,6 +13,7 @@ describe("Profile Page - Accessibility Test", () => {
     cy.authenticate("stateUser");
 
     cy.visit("/profile");
+    cy.wait(1000);
```

Comment:
Can we wait for something instead of waiting for 1000 ms?

Reply:
I'd love to 😭 cypress doesn't have many options for that though. I tried awaiting an api call that should happen on that page but it still gets the error. not sure what is causing it.

Reply:
Ah.  I haven't done much with cypress.  I've never tried this, but might be worth taking a look https://www.npmjs.com/package/cypress-wait-until

Reply:
@gmrabian you might be able to replace this with a `cy.get("h1")` (could replace with another selector like "main"). I think this will ensure that the page has finished rendering before making the a11y assertions.

Reply:
Changed and it worked!